### ISSUE TITLE
convo.addQuestion fails silently when using 'blocks'

### ISSUE THREAD

**Issue by jeffmarshall** [howdyai/botkit#1687]
### What are you trying to achieve or the steps to reproduce?

I tried to use Slack's [block kit](https://api.slack.com/block-kit) blocks with `convo.addQuestion`.

````javascript
convo.addQuestion({
  blocks: [
    {
      type: 'section',
      text: {
        type: 'plain_text',
        text: 'This is a plain text section block.',
        emoji: true
      }
    }
  ]
}, [
  // ... pattern, callback
], { key: 'foo' }, 'thread_name');
````

### What was the result you received?

Nothing happens.

### What did you expect?

I expected to see a message posted in slack containing a section block with the words "This is a plain text section block.", or at least an error message in the stdout. I believe I'm attempting to use blocks in the right way.

### Context:

- Botkit version:
- Messaging Platform: Slack
- Node version: 10.15.3
- Os: MacOS 10.14

**Comment by jeffmarshall** [comment]
I think the problem is in `botkit/lib/CoreBot.js` on line 975 inside the function `this.tick`:

````
if (message.text || message.attachments || message.attachment) {
  // do some stuff, then send the message
}
````

Messages with blocks don't need to have text at the top level of the message payload, so the message fails this condition. If I add `text: 'foo'` to my original message, then the message is sent normally and the blocks appear as they should in slack. I am afraid that this is only a workaround. The real problem may lie in this condition.

I tried to make a pull request, but I can't find where this code comes from in botkit to edit it.

**Comment by benbrown** [comment]
This is fixed in Botkit 4.5
